control/passivity.py

```diff
@@ -0,0 +1,65 @@
+'''
+Author: Mark Yeatman  
+Date: May 15, 2022
+'''
+
+import numpy as np
+
+try:
+    import cvxopt as cvx
+except ImportError as e:
+    cvx = None
+
+def is_passive(sys):
+    '''
+    Indicates if a linear time invarient system is passive
+
+    Constructs a linear matrix inequality and a feasibility optimization
+    such that is a solution exists, the system is passive.
+
+    The source for the algorithm is: 
+    McCourt, Michael J., and Panos J. Antsaklis. "Demonstrating passivity and dissipativity using computational methods." ISIS 8 (2013).
+    '''
+    if cvx is None:
+        raise ModuleNotFoundError
```

Comment:
You might want to add a message to this exception so that the user knows what the need to do (eg, "cvxopt required for passivity module").

Reply:
Change taken.

Reply:
The output needs to be tied together. Print going to `stdout` might end up somewhere else than the backtrace (e.g. the exception and output captures by pytest):

```suggestion
        raise ModuleNotFoundError("cvxopt required for passivity module")
```

```
[ben@skylab:~]% python3 -c 'raise  ModuleNotFoundError("cvxopt required for passivity module")'                                            [0]
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: cvxopt required for passivity module
[ben@skylab:~]% python3 -c 'print("cvxopt required for passivity module"); raise ModuleNotFoundError'                                      [1]
cvxopt required for passivity module
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError
```

Reply:
GitHub wouldn't let me suggest the removal of the print line in the same comment thread. Please remove the duplicate.

Reply:
Will do